itens/views.py

```diff
@@ -40,9 +40,10 @@ def get(self, request, *args, **kwargs):
             user_is_authenticated = request.user.is_authenticated()
```

Comment:
https://docs.djangoproject.com/en/2.2/topics/auth/default/#the-login-required-decorator